src/taco-cli/resources/en/resources.json

```diff
@@ -378,6 +378,8 @@
     "_CommandRunEmulatorDescription.comment": "Description of the --emualtor argument to 'taco run'",
     "CommandRunTargetDescription": "Run a build on the specified target",
     "_CommandRunTargetDescription.comment": "Description of the --target argument to 'taco run'",
+    "CommandRunListDescription": "Report possible targets to run on without actually running",
```

Comment:
Additionally, specify that both both device and emulator targets will be listed (unless DEVICECLASS option is specified).